src/platform/APM32/platform_mcu.h

```diff
@@ -199,8 +199,7 @@
 
 #define DMA_TRAIT_CHANNEL 1
 
-#endif
```

Comment:
leftover ?

Reply:
line 204 is removed as well, so it just extends the 1 `#if`, AFAICT

Reply:
Correct... same #if was repeated